src/pages/objkt-display/styles.module.scss

```diff
@@ -119,124 +96,120 @@
 .history__ed,
 .history__price,
 .history__date {
-  margin-bottom: 7px;
-
-  height: 20px;
-  white-space: nowrap;
-
-  @include respond-to('desktop') {
-    margin-bottom: 0;
-    overflow: hidden;
-    text-overflow: clip;
-  }
+    margin-bottom: 7px;
+    height: 20px;
+    white-space: nowrap;
+    @include respond-to('desktop') {
+        margin-bottom: 0;
+        overflow: hidden;
+        text-overflow: clip;
+    }
 }
 
 .history__date {
-  overflow: visible;
+    overflow: visible;
 }
 
 .history__event {
-  @include respond-to('desktop') {
-    width: calc(9.166666666%);
-  }
+    @include respond-to('desktop') {
+        width: calc(9.166666666%);
+    }
 }
 
 .history__from,
 .history__to {
-  @include respond-to('desktop') {
-    text-overflow: ellipsis;
-    width: calc(20.833333333%);
-  }
+    @include respond-to('desktop') {
+        text-overflow: ellipsis;
+        width: calc(20.833333333%);
+    }
 }
 
 .history__ed {
-  @include respond-to('desktop') {
-    text-align: center;
-    width: calc(6.944444444%);
-  }
+    @include respond-to('desktop') {
+        text-align: center;
+        width: calc(6.944444444%);
+    }
 }
 
 .history__price,
 .history__date {
-  @include respond-to('desktop') {
-    text-align: right;
-    width: calc(13.888888888%);
-  }
+    @include respond-to('desktop') {
+        text-align: right;
+        width: calc(13.888888888%);
+    }
 }
 
 .history__royalties {
-  font-size: $font-small-mobile;
-
-  @include respond-to('desktop') {
-    font-size: $font-small;
-    margin-top: 20px;
-    padding-top: 10px;
-
-    width: 33.333333333%;
-    border-top: 1px solid var(--border-color);
-  }
+    font-size: $font-small-mobile;
+    @include respond-to('desktop') {
+        font-size: $font-small;
+        margin-top: 20px;
+        padding-top: 10px;
+        width: 33.333333333%;
+        border-top: 1px solid var(--border-color);
+    }
 }
 
 .history__secondary {
-  color: var(--secondary-color);
-  margin-right: 1em;
+    color: var(--secondary-color);
+    margin-right: 1em;
 }
 
 .history__mobile {
-  display: block;
-
-  @include respond-to('desktop') {
-    display: none;
-  }
+    display: block;
+    @include respond-to('desktop') {
+        display: none;
+    }
 }
 
 .history__from,
 .history__to {
-  display: flex;
+    display: flex;
 }
 
 .history__ed,
 .history__price,
 .history__date {
-  display: none;
-
-  @include respond-to('desktop') {
-    display: block;
-  }
+    display: none;
+    @include respond-to('desktop') {
+        display: block;
+    }
 }
 
 .history__inner__mobile {
-  display: flex;
-  flex-direction: column;
-
-  text-align: right;
-
-  .history__ed,
-  .history__price,
-  .history__date {
-    display: block;
-  }
-
-  @include respond-to('desktop') {
-    display: none;
-  }
+    display: flex;
```

Comment:
Did husky worked on your end? There is a lot of style edits

Reply:
Wasn't aware of the husky step. Don't think I did anything like that before. What do I do now?

Reply:
It should autoinstall on npm install (the prepare script runs just after) the first time and its job is to run cmd on git events, we use it for commits, it lint/formats staged files to avoid this. I will check it maybe it actually needed those as it's only ran on staged files, but we ran it over the whole codebase once when we introduced husky so not sure.

Reply:
I'm not sure if it was automatically installed. Never saw any husky errors or anything that looked like a pre-commit trigger. This is what I got with the prepare target:
npm run prepare

> teia-ui@0.1.0 prepare
> husky install

husky - Git hooks installed

Reply:
This should mean it's properly running, I'll test locally from main making the same edits to see the behaviour on my end